### PR TITLE
docs: add KB article for Groups

### PR DESCRIPTION
**Pull request by furi0us333**
- Add KB article for Groups to better explain the use case
- Add link to Phylum UI in Quickstart/Readme

### CODE REVIEW

**Comment by kylewillmon** [comment]
Are we adding KB articles to the CLI docs now? Seems like that would be better placed somewhere else.

File: docs/phylum.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Interact with the Phylum API
 * [phylum project](https://docs.phylum.io/docs/phylum_project)
 
 ### Additional Commands
+* [phylum group](https://docs.phylum.io/docs/phylum_group)
```
Comment (kylewillmon) [inline docs/phylum.md:15]: Where is this file?

**Review by kylewillmon — CHANGES_REQUESTED** [review]
I think a file is missing

**Comment by furi0us333** [comment]
> Are we adding KB articles to the CLI docs now? Seems like that would be better placed somewhere else.

You will notice the `category` in the front matter is slightly different on `groups.md` which will place it in the KB section of the docs. In terms of docs as code, they just happen to live in this repo.

File: docs/phylum.md (hunk 1 above)
Comment (furi0us333) [reply docs/phylum.md:15]: https://github.com/phylum-dev/cli/blob/main/docs/phylum_group.md

It is new and won't be live until we release.

**Comment by kylewillmon** [comment]
> they just happen to live in this repo.

Ah. I didn't notice that we already have a few of these... But we don't have all of them (i.e., [Phylum Package Score](https://docs.phylum.io/docs/phylum-package-score) and [Processing](https://docs.phylum.io/docs/processing) aren't here)...

I think these files should find a new home...

File: docs/phylum.md (hunk 1 above)
Comment (kylewillmon) [reply docs/phylum.md:15]: Oh... They already existed, but weren't linked. My bad.

**Pull request merged**